src/components/Pill/PillColors.ts

```diff
@@ -0,0 +1,28 @@
+export const PillColors = {
+  pill: {
+    gray: '#E8E8E8',
+    blue: '#BFCBE8',
+    turquoise: '#CCE9FB',
+    teal: '#BFDDDD',
+    green: '#C7E6D0',
+    yellow: '#FEF6CC',
+    orange: '#FFDEBF',
+    red: '#FBBFBF',
+    pink: '#F3D7F9',
+    purple: '#D9CFFC',
+    brown: '#E2CDBF',
+  },
+  pillHover: {
+    gray: '#D8D8D8',
+    blue: '#A6B6DF',
+    turquoise: '#B8DFFA',
+    teal: '#A6CFCF',
+    green: '#B1DCBE',
+    yellow: '#FDF2B8',
+    orange: '#FFD0A6',
+    red: '#FAA6A6',
+    pink: '#EEC8F6',
+    purple: '#C9BCFB',
+    brown: '#D6B9A6',
+  },
```

Comment:
This looks like lot of repetition, can we simplify the format to:
```js
const PillColors = {
  gray: {default: '#E8E8E8', hover: '#D8D8D8' },
  blue: { default: ..., hover: ... },
  turquoise: ...,
```
or maybe even using tuple would be enough like
```js
const PillColors = {
  gray: ['#E8E8E8', '#D8D8D8'],
  blue: [default, hover],
  turquoise: ...,
```